.changeset/red-wolves-greet.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': patch
+---
+
+In the base table with sortable columns, the direction symbol was too close to the header text in condensed mode. This change increases left padding on TableCell and TableHeader for condensedStyle.
```

Comment:
Changesets should be concise and focus on the _what_, not the _how_. Think about it this way: what information is relevant to a developer who is reading the changelog before upgrading Circuit UI in their application?

`TableCell` and `TableHeader` are internal components that the developer wouldn't know about. The padding is an implementation detail.

```suggestion
Fixed a spacing issue in the `Table` component when sorting a column in `condensed` mode.
```

Reply:
Thanks for the nice comment! Makes a lot of sense! Committed the suggested change. :)